cmd/pebble/cmd_help_test.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main_test
+
+import (
+	"github.com/jessevdk/go-flags"
+	. "gopkg.in/check.v1"
+
+	pebble "github.com/canonical/pebble/cmd/pebble"
+)
+
+func (s *PebbleSuite) TestHelpCommandWorks(c *C) {
+	rest, err := pebble.Parser(pebble.Client()).ParseArgs([]string{"help", "add"})
+
+	flagsErr, ok := err.(*flags.Error)
+	c.Assert(ok, Equals, true)
+	c.Assert(flagsErr.Type, Equals, flags.ErrHelp)
+
+	c.Assert(rest, HasLen, 0)
+	c.Assert(s.Stdout(), Equals, "")
```

Comment:
It was surprising to me that stdout was empty here, because it should be printing a help message. Then I looked at the code and realized the `help foo` case (and some other help cases) are handled in `cmd/pebble/main.go:run`.

So I think for the help tests we should change the tests to call run(). You can do this by calling `RunMain` which is defined to be `run` in `export_test.go` (see `TestErrorResult` for an example). This also uses `fakeArgs()` -- which IMO should be done as an argument to `run()` like `run(args []string)` explicitly, so feel free to change that and get rid of `fakeArgs` if you want.

Then we can test that `s.Stdout()` actually has something in it. I think we shouldn't test for the entire help text -- that's annoying to maintain -- but that a couple of small snippets are present, for example `Usage:` and `pebble add` and `The add command` or something.